### PR TITLE
[CMS] Use usernames instead of uuids for supervisors/subcontractors

### LINKED ISSUE #2064: [cms] Show sub contractor username instead of ID of sub contractor line items
Currently the userid is shown when selected in the subcontractor dropdown.  Instead it should be displayed as their username in invoice details.

### LINKED ISSUE #2066: [cms] Only show 'sub-contractor' as an option for DCC when user is supervisor
Currently Direct and Subcontractor are shown on the Submit DCC form, but really sub-contractor should only be shown as an option if the currently logged in user is a supervisor user type.

### PR DESCRIPTION
**Pull request by thi4go**
This PR changes some information display of supervisors/subcontractors. Instead of showing their uuid, this diff shows the username instead. Places changed:

- Invoice Detail
- Viewing account details with supervisors field filled

Closes #2064

Another change related to this domain is in the DCC form, where now a user can only issue a DCC with subcontractor type when the logged in user requesting the DCC is of type supervisor. So only supervisors can issue a DCC with the subcontractor type.

closes #2066

### CODE REVIEW

**Comment by alexlyp** [comment]
Needs a rebase.

**Comment by thi4go** [comment]
Rebased mate @alexlyp

File: src/components/DccForm/DccForm.jsx [hunk 1]
```diff
@@ -16,9 +16,11 @@ import {
   getNomineeOptions,
   DCC_TYPE_ISSUANCE,
   DCC_TYPE_REVOCATION,
-  CONTRACTOR_TYPE_REVOKED
+  CONTRACTOR_TYPE_REVOKED,
+  CONTRACTOR_TYPE_SUPERVISOR
 } from "src/containers/DCC";
 import usePolicy from "src/hooks/api/usePolicy";
```
Comment (amassarwi) [inline src/components/DccForm/DccForm.jsx:22]: We could add a `index.js` file to `src/hooks/api` to be able to multi import hooks here and everywhere.

**Comment by thi4go** [comment]
Hey mate @amassarwi really liked your suggestion! Added the `index.js` file to our `src/hooks` folder so that we can easily import hooks from both `api` and `utils`. It's ready for another review. Thanks!

File: src/hooks/index.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+export { default as useModalContext } from "./utils/useModalContext";
+export { default as useOnboardModal } from "./utils/useOnboardModal";
+export { default as useOnRouteChange } from "./utils/useOnRouteChange";
+export { default as useQueryString } from "./utils/useQueryString";
+export { default as useQueryStringWithIndexValue } from "./utils/useQueryStringWithIndexValue";
+export { default as useScrollFormOnError } from "./utils/useScrollFormOnError";
+export { default as useScrollToTop } from "./utils/useScrollToTop";
+export { default as useSessionStorage } from "./utils/useSessionStorage";
+export { default as useThrowError } from "./utils/useThrowError";
+export { default as useWhyDidYouUpdate } from "./utils/useWhyDidYouUpdate";
```
Comment (victorgcramos) [inline src/hooks/index.js:35]: Good job here! 🚀

**Pull request merged**